client/my-sites/plans/main.jsx

```diff
@@ -95,14 +87,8 @@ const Plans = React.createClass( {
 } );
 
 export default connect(
-	( state ) => {
-		const selectedSiteId = getSelectedSiteId( state );
-		const isPlaceholder = ! selectedSiteId;
-		return {
-			isPlaceholder,
-			plans: getPlans( state ),
-			selectedSite: getSelectedSite( state ),
-			selectedSiteId: selectedSiteId
-		};
-	}
+	( state ) => ( {
+		isPlaceholder: ! getSelectedSiteId( state ),
+		selectedSite: getSelectedSite( state )
```

Comment:
Since `getSelectedSite` depends on `getSelectedSiteId` I think we can simplify these two props with:
```js
const selectedSite = getSelectedSite( state );
return {
    isPlaceholder: ! selectedSite,
    selectedSite
};
```